tests/transforms/test_qcut.py

```diff
@@ -1908,6 +1908,52 @@ def circuit(x):
 
         assert np.isclose(grad, grad_expected)
 
+    def test_simple_cut_circuit_torch_jit(self, mocker, use_opt_einsum):
+        """
+        Tests the full circuit cutting pipeline returns the correct value and
+        gradient for a simple circuit using the `cut_circuit` transform with the torch interface and
+        using JIT.
+        """
+        torch = pytest.importorskip("torch")
+
+        dev = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev, interface="torch")
+        def circuit(x):
+            qml.RX(x, wires=0)
+            qml.RY(0.543, wires=1)
+            qml.WireCut(wires=0)
+            qml.CNOT(wires=[0, 1])
+            qml.RZ(0.240, wires=0)
+            qml.RZ(0.133, wires=1)
+            return qml.expval(qml.PauliZ(wires=[0]))
+
+        x = torch.tensor(0.531, requires_grad=True)
+
+        # Note that the jit.trace ends up calling qcut_processing_fn multiple times, so below we
+        # delay introducing the spy until afterwards and then ensure that qcut_processing_fn is
+        # not called again.
+        cut_circuit_jit = torch.jit.trace(
+            qcut.cut_circuit(circuit, use_opt_einsum=use_opt_einsum), x
+        )
```

Comment:
@trbromley very very minor but I'm not sure `torch.jit.trace` is actually JITing. I think `torch.jit.script` is the decorator that actually performs compilation (but also don't think it works with PL).

Having said that, tracing is still good to test and still faster than non-tracing.

Reply:
Thanks @josh146! I reworded here: 9bf8d0de7817ff0f1d8048f3cfd8bdf4b7d8b522. Couldn't get scripting to work!